source/chapters/user_interface.rst

```diff
@@ -135,8 +138,9 @@ separate waveforms. Select your preferred appearance in
    Mixxx alternative skin (Deere) - Separate waveforms
 
 .. warning :: If you have a slower computer and notice performance issues with
-              Mixxx, try to lower the frame rate or change the waveform type in
-              :menuselection:`Preferences --> Waveforms`.
+              Mixxx, try to lower the frame rate or change the type for waveform
```

Comment:
Flows a little better?

"try lowering the frame rate or changing the waveform type in :menuselection:`Preferences --> Interface --> Waveforms"